styles/musicians.css

```diff
@@ -0,0 +1,98 @@
+.featuredMusicians {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 20px;
+  padding-bottom: 40px;
+}
+
+.director {
+  font-size: 22px;
+  padding-bottom: 10px;
+}
+
+.cardsDiv {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 90%;
+  justify-content: space-around;
+}
+
+.musicianCard {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+  margin-left: 10px;
+  margin-right: 10px;
+  height: 150px;
+}
```

Comment:
- [ ] Great job implementing the musician's section, it looks almost identical to the provided design :clap: :sunglasses: However the cards overlap when clicking on the `MORE` button. Kindly check it to make your website more professional.
### Your design :point_down: 
![Screenshot from 2023-06-29 01-38-09](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/c1ba7ac4-40e9-41ba-9a1e-51a1f64c4178)